apps/prairielearn/src/lib/externalGraderLocal.ts

```diff
@@ -205,15 +225,15 @@ export class ExternalGraderLocal {
  *
  * On Windows, we use $USERPROFILE instead of $HOME.
```

Comment:
This comment seems to misleadingly assume that this code might run on Windows. I don't think that's possible, there are a lot of places in the codebase that assume a POSIX environment.

Reply:
Please remove this comment!